core/templates/dev/head/components/review-material-editor/review-material-editor.directive.html

```diff
@@ -0,0 +1,47 @@
+<div ng-if="!$ctrl.conceptCardExplanationEditorIsShown" class="review-material" style="padding-top: 0.5em;">
+  <div ng-class="oppia-editable-section"
+       ng-click="$ctrl.openConceptCardExplanationEditor()">
+    <i class="material-icons oppia-editor-edit-icon float-right protractor-test-edit-concept-card"
+       title="Edit Concept Card Explanation">&#xE254;
+    </i>
+    <div class="oppia-state-content-display oppia-transition-200 protractor-test-concept-card-explanation"
+         ng-class="oppia-prevent-selection"
+         title="Concept Card Explanation">
+      <span class="oppia-placeholder" ng-show="$ctrl.editableExplanation === ''">
+          Give a review or explanation of the skill.
+      </span>
+      <angular-html-bind html-data="$ctrl.getBindableDict().displayedConceptCardExplanation">
+      </angular-html-bind>
+    </div>
+    <!-- This is a dummy div created to mask the contents when the user hovers over the content. -->
+    <div class="oppia-editable-section-mask">
+    </div>
+  </div>
+</div>
+
+<div ng-if="$ctrl.conceptCardExplanationEditorIsShown">
+  <schema-based-editor schema="$ctrl.HTML_SCHEMA" class="protractor-test-concept-card-text" local-value="$ctrl.editableExplanation">
+  </schema-based-editor>
+  <div class="editor-buttons">
+    <button type="button"
+            class="btn btn-success oppia-save-state-item-button float-right protractor-test-save-concept-card"
+            ng-click="$ctrl.saveConceptCardExplanation()">
+      Save
+    </button>
+    <button type="button" class="btn btn-secondary float-right" ng-click="$ctrl.closeConceptCardExplanationEditor()">Cancel</button>
+    <div style="clear: both;"></div>
+  </div>
+</div>
+<style>
+  review-material-editor .review-material {
+   padding-bottom: 36px;
```

Comment:
Indent by 2 from the previous line. Ditto below.

Reply:
Done